### PR TITLE
[Doc FR]: `guides/*.md` translated.

### PR DESCRIPTION
**Pull request by Haeresis**
Hi ! We are Vuejs-FR team and we translated all Vue documentation in french. We have finish to translate the Guides part of vue-test-utils documentation.

We also have added a message in API page not still translated to invite anyone want to help us with the translation.

Feel free to merge this PR now (really useful because all part already translated will be availlable or wait we have translate all API part too.

### CODE REVIEW

**Comment by eddyerburgh** [comment]
I'll wait until the API is translated. Looks good so far 😀

**Comment by Haeresis** [comment]
OK!

**Comment by eddyerburgh** [comment]
@posva @Akryum  Would either of you be able to review this PR?

**Comment by Haeresis** [comment]
Just focus on files under `guides/`, I will add « english » mention in others files before merge.

File: docs/fr/SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+## Table of Contents
```
Comment (posva) [inline docs/fr/SUMMARY.md]: Je penseque ça serait bien d'avoir le fichier entier traduit

File: docs/fr/SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+* [Guides](guides/README.md)
+  * [Commencer](guides/getting-started.md)
+  * [Astuces](guides/common-tips.md)
+  * [Mouse, Key and other DOM Events (EN)](guides/dom-events.md)
```
Comment (posva) [inline docs/fr/SUMMARY.md]: idem

File: docs/fr/guides/choosing-a-test-runner.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Choisir un lanceur de tests
+
+Un lanceur de tests est un programme qui lance des tests (et oui !).
+
+Il y a un nombre conséquent de lanceurs de tests JavaScript et `vue-test-utils` fonctionne avec tous. `vue-test-utils` est agnostique en termne de lanceurs de tests.
```
Comment (posva) [inline docs/fr/guides/choosing-a-test-runner.md]: terme

File: docs/fr/guides/common-tips.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+Le bénéfice de cette approche est que tant que l'interface public de votre composant reste la même, vos tests passeront et ce peu importe le comportement interne de votre composant, qui pourrait changer avec le temps.
+
+Ce sujet est discuté plus en détails dans une [très bonne présentation de Matt O'Connell](http://slides.com/mattoconnell/deck#/).
+
+## Rendu superficiel
+
+Dans des tests unitaires, on souhaite s'intéresser au composant qui est en train d'être testé comme une unité isolée et ainsi éviter de s'assurer du bon comportement des composants enfants.
+
+De plus, pour les composants qui contiennent beaucoup de composants enfant, l'intégralité de l'arbre de rendu peut être énorme. Répétitivement rendre tous les composants pourrait réduire la vitesse de nos tests.
```
Comment (posva) [inline docs/fr/guides/common-tips.md]: composants enfants, non?

File: docs/fr/guides/testing-SFCs-with-mocha-webpack.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+- Le paramètre `--require` permet de s'assurer que le fichier `test/setup.js` est bien exécuté avant les tests. Dans celui-ci, on met en place l'environnement où nos tests vont être exécutés.
+
+- Le dernier paramètre est un glob pour indiquer les fichiers de tests à inclure dans le paquetage.
+
+### Configuration supplémentaire pour webpack
+
+#### Externaliser les dépendances npm
+
+Dans nos tests, nous allons surement importer un nombre conséquent de dépendances npm, certaines d'entre elles n'ont pas été conçues pour une utilisation dans un navigateur et ne peuvent être empaquetées par webpack. Il faut aussi considérer qu'externaliser les dépendances augmente énormément la vitesse de lancement des tests. On peut externaliser toutes les dépendances npm avec `webpack-node-externals -externals` :
```
Comment (posva) [inline docs/fr/guides/testing-SFCs-with-mocha-webpack.md]: un petit ` -externals` en trop

File: docs/fr/guides/using-with-vue-router.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  path: '/un/super/chemin'
+}
+
+const wrapper = shallow(Component, {
+  mocks: {
+    $route
+  }
+})
+
+wrapper.vm.$router // /un/super/chemin
```
Comment (posva) [inline docs/fr/guides/using-with-vue-router.md]: @eddyerburgh I think there's a typo here, shouldn't et be `wrapper.vm.$route.path`?

Comment (eddyerburgh) [reply docs/fr/guides/using-with-vue-router.md]: Yep you're correct 😀

**Comment by Haeresis** [comment]
Thanks a lot to you @posva ;)

Guide part up to date. We contunue api and new components part.

**Comment by eddyerburgh** [comment]
Great work guys 🙌

**Comment by Haeresis** [comment]
\o/

**Pull request merged**